internal/processor/filterspan/filterspan.go

```diff
@@ -121,35 +156,104 @@ func newAttributesMatcher(mp *filterconfig.MatchProperties) (attributesMatcher,
 			if err != nil {
 				return nil, err
 			}
-			entry.AttributeValue = &val
+
+			if config.MatchType == filterset.Regexp {
```

Comment:
Does this add support for Regexp matching? It appears previously we were returning an error if match type was not Strict.

Reply:
yes, indeed - I've updated the ticket description accordingly